src/sink.cpp

```diff
@@ -69,7 +69,26 @@ void Sink::subscribe(const std::vector<std::string>& metrics, int64_t expires)
 void Sink::sink_config(const json& config)
 {
     data_config(config);
-    data_queue_ = config.at("dataQueue");
```

Comment:
Don't do that, we should fix the manager instead.

Reply:
This should be fixed since metricq/metricq-manager@988a92b0a477033b77e502c5bd3dc42ded50c8a4, shouldn't it?

Other than that the check only makes sure that the dataQueue returned from the manager is indeed the same as the one configured locally, if it was indeed already configured.

Reply:
It's a wanted feature that they can be different